myJava.js

```diff
@@ -0,0 +1,95 @@
+function myFunction(){
+		var verb = document.getElementById("verbs").value;
+		var noun = document.getElementById("nouns").value;
+		var adverb = document.getElementById("adverbs").value;
+		var adjective = document.getElementById("adjectives").value;
+
+		var verbArr = verb.split(",");
+		var nounArr = noun.split(",");
+		var adverbArr = adverb.split(",");
+		var adjectiveArr = adjective.split(",");
+
+		function makeBold(arr){
+			return arr.map(function (item){
+				return "<strong>" + item + "</strong>";
+			});
+		}
+
+		function emptyString(arr){
+			for(var i=arr.length-1; i > -1 ; i--){
+				if(arr[i] === ""){
+					arr.splice(i,1);
+				}
+			}
+		}
+
+		emptyString(verbArr);
+		emptyString(nounArr);
+		emptyString(adverbArr);
+		emptyString(adjectiveArr);
+
+		if(verbArr.length < 4){
+			while(verbArr.length < 4){
```

Comment:
This while loop inside the if statement is interesting. You actually don't need the if statement, it isn't "wrong" but its interesting to think about why you don't need.